### DISCUSSION TITLE
is it possible to make a card like this?

### DISCUSSION THREAD

**Discussion by edwardtich1** [custom-cards/button-card#858] [Ideas]
Hi
is it possible to make a card like this?
![2024-07-19_14-44-40](https://github.com/user-attachments/assets/8763084f-5d00-4beb-bc29-ae9509fb1337)

**Comment by Mariusthvdb** [comment]
No

this is possible
<img width="497" alt="Scherm­afbeelding 2024-07-22 om 17 02 07" src="https://github.com/user-attachments/assets/a1937c3e-f3a3-4d1d-873c-e6daeada727f">
